class.jetpack.php

```diff
@@ -4776,11 +4776,25 @@ public static function delete_secrets( $action, $user_id ) {
 	 * @since 2.6
 	 * @return int
 	 **/
-	public function get_remote_query_timeout_limit() {
+	public static function get_max_execution_time() {
 	    $timeout = (int) ini_get( 'max_execution_time' );
 	    if ( ! $timeout ) // Ensure exec time set in php.ini
-				$timeout = 30;
-	    return intval( $timeout / 2 );
+			$timeout = 30;
```

Comment:
Also, there are leading spaces before all lines except one in this method. That's a minor nitpick, but should be an easy fix if you go back in.